### PR TITLE
Add pre verification gas to chain gas constants

### PR DESCRIPTION
**Pull request by nalinbhardwaj**

### CODE REVIEW

File: apps/daimo-mobile/src/model/account.ts [hunk 1]
```diff
@@ -71,6 +71,14 @@ export function toEAccount(account: Account): EAccount {
   return { addr: account.address, name: account.name };
 }
 
+// Pre-v9 chain gas constants
+type OldChainGasConstants = {
```
Comment (dcposch) [inline apps/daimo-mobile/src/model/account.ts:75]: `ChainGasConstantsV8`?

File: apps/daimo-mobile/src/model/account.ts [hunk 2]
```diff
@@ -220,7 +228,7 @@ export function parseAccount(accountJSON?: string): Account | null {
       pendingNotes: a.pendingNotes || [],
       pendingKeyRotation: [],
 
-      chainGasConstants: a.chainGasConstants,
+      chainGasConstants: { ...a.chainGasConstants, preVerificationGas: "0" },
```
Comment (dcposch) [inline apps/daimo-mobile/src/model/account.ts:231]: 👍

File: packages/daimo-api/src/contract/paymaster.ts [hunk 3]
```diff
@@ -89,7 +103,47 @@ export class Paymaster {
...
   }
 }
+
+type DummyAddDeviceOps = {
+  base: UserOpHex;
+  baseGoerli: UserOpHex;
+};
+
+// Arbitrary add device ops used for gas estimation via Pimlico.
+const dummyAddDeviceOps: DummyAddDeviceOps = {
```
Comment (dcposch) [inline packages/daimo-api/src/contract/paymaster.ts:118]: Nice, I love the simplicity of just a hardcoded reference op

**Review by dcposch — APPROVED** [review]
LGreatTM

File: apps/daimo-mobile/src/model/account.ts (hunk 1 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/model/account.ts]: done

**Pull request merged**